sway/commands/move.c

```diff
@@ -747,6 +748,38 @@ static struct cmd_results *cmd_move_in_direction(
 	return cmd_results_new(CMD_SUCCESS, NULL);
 }
 
+static struct cmd_results *cmd_move_to_position_pointer(struct sway_container *container) {
+	struct sway_seat *seat = config->handler_context.seat;
+	if (!seat->cursor) {
+		return cmd_results_new(CMD_FAILURE, "No cursor device");
+	}
+	struct wlr_cursor *cursor = seat->cursor->cursor;
+	/* Determine where to put the window. */
+	double lx = cursor->x - container->width / 2;
+	double ly = cursor->y - container->height / 2;
+
+	/* Correct target coordinates to be in bounds (on screen). */
+	for (int i = 0; i < root->outputs->length; ++i) {
```

Comment:
I was unable to find a better way to get the output from the cursor, if there is a better way, I'd be happy to change this. (Maybe add a function to wlroots to get the output from the cursor's state?)